README.md

```diff
@@ -73,7 +73,7 @@ Once the Onfleet object is created, you will get access to all the API endpoints
 Run `npm test`
 
 ### Throttling
-Rate limiting is enforced by the API with a threshold of 20 requests per second across all your organization's API keys, learn more about it [here](https://docs.onfleet.com/reference#throttling). 
+Rate limiting is enforced by the API with a threshold of 20 requests per second across all your organization's API keys, learn more about it [here](https://docs.onfleet.com/reference#throttling). We have implemented a limiter on the wrapper itself to avoid you from excessively hitting Onfleet's rate limitations and eventually be banned for.
```

Comment:
I would change the copy to `unintentionally exceeding your rate limit`

Reply:
obv, have to do the same for all languages

Reply:
Language updated in https://github.com/onfleet/node-onfleet/pull/14/commits/753b8526032f72d1f80c2ad6e3930d78165a7984